src/towncrier/test/test_write.py

```diff
@@ -109,6 +109,7 @@ def test_append_at_top(self):
                 wrap=True,
                 versiondata={"name": "MyProject", "version": "1.0", "date": "never"},
             ),
+            True,
```

Comment:
I think that using keyword arguments makes the test a bit easier to read.

```suggestion
            single_file=True,
```